.github/dependabot.yml

```diff
@@ -9,7 +9,15 @@ updates:
       time: "09:00"
       timezone: "America/Los_Angeles"
     open-pull-requests-limit: 30
-  
+
+  - package-ecosystem: "nuget"
+    directory: "/converter/dicom-cast"
```

Comment:
I can't seem to find much documentation on this. Am I suppose to specify the solution directory?

Reply:
I believe that is the case.